include/xtensor/xstridedview.hpp

```diff
@@ -754,10 +756,10 @@ namespace xt
      * sv.push_back(xt::all());
      * \endcode
      */
-    class slice_vector : private std::vector<std::array<long int, 3>>
+    class slice_vector : public std::vector<std::array<long int, 3>>
```

Comment:
Othwewise the `using pop_back`  does not actually enable the method as public API.

Reply:
That's strange, because it works for the other methods (`operator[]`, `push_back`, etc). If the inheritance is public, no need for the using declarations anymore, but that's not really a good practice to publicly inherit from `std::vector`.

Reply:
Private inheritance + using declaration for pop_back work fine for me with Visual Sutio and gcc 5, so I think the problem is due to something else.

Reply:
Will check again when I am closer to a keyboard.